src/cpp/server/backend_metric_recorder.cc

```diff
@@ -46,18 +46,14 @@ std::unique_ptr<ServerMetricRecorder> ServerMetricRecorder::Create() {
   return std::unique_ptr<ServerMetricRecorder>(new ServerMetricRecorder());
 }
 
-ServerMetricRecorder::ServerMetricRecorder() {
-  // Starts with an empty result.
-  metric_state_ =
-      std::make_shared<const ServerMetricRecorder::BackendMetricDataState>();
-}
+ServerMetricRecorder::ServerMetricRecorder()
+    : metric_state_(std::make_shared<
+                    const ServerMetricRecorder::BackendMetricDataState>()) {}
```

Comment:
No need for `ServerMetricRecorder::` prefix here either.

Reply:
Done